src/test/kotlin/no/nav/yrkesskade/saksbehandling/repository/BehandlingRepositoryTest.kt

```diff
@@ -65,4 +65,94 @@ class BehandlingRepositoryTest : AbstractTest() {
         val behandlinger = behandlingRepository.findBySaksbehandlingsansvarligIdent("test", Pageable.ofSize(10))
         assertThat(behandlinger.size).isEqualTo(0)
     }
+
+    @Test
+    fun `hent aapne behandlinger med alle filter`() {
+        // given
+        val sak = sakRepository.findAll().first()
+        val ikkePaabegyntBehandling = genererBehandling(1L, null, Behandlingsstatus.IKKE_PAABEGYNT, sak)
+        behandlingRepository.save(ikkePaabegyntBehandling)
+        val underBehandlingBehandling = genererBehandling(2L, "test", Behandlingsstatus.UNDER_BEHANDLING, sak)
+        behandlingRepository.save(underBehandlingBehandling)
+        val ferdigBehandling = genererBehandling(3L, "test", Behandlingsstatus.FERDIG, sak)
+        behandlingRepository.save(ferdigBehandling)
+
+        //when
+        val behandlinger = behandlingRepository.findBehandlingerBegrensetTilBehandlingsstatuser(Behandlingsstatus.UNDER_BEHANDLING, "enFinKategori", Behandlingstype.VEILEDNING, listOf(Behandlingsstatus.UNDER_BEHANDLING, Behandlingsstatus.IKKE_PAABEGYNT), Pageable.unpaged())
+
+        // then
+        assertThat(behandlinger.size).isEqualTo(1)
```

Comment:
Skulle ikke svaret vært 2 (ikke påbegynt og under berhandling)? Eller er det filtrerering på saksbeh.ansv.ident?

Reply:
Nei, vi vil kun ha de som har status UNDER_BEHANDLING. Dersom første parameter er null (tomt filter), ville svaret vært to.